en/deploy/README.md

```diff
@@ -4,243 +4,301 @@
+# Git
 
-    (myvenv) $ pip install dj-database-url gunicorn whitenoise
+Git is a "version control system" used by a lot of programmers - software which keeps track of changes to a file or set of files over time so that you can recall specific versions later. 
 
-After the installation is finished, go to the `djangogirls` directory and run this command:
 
-    (myvenv) $ pip freeze > requirements.txt
+## Installing Git
 
-This will create a file called `requirements.txt` with a list of your installed packages (i.e. Python libraries that you are using, for example Django :)).
+### Windows
 
-> __Note__: `pip freeze` outputs a list of all the Python libraries installed in your virtualenv, and the `>` takes the output of `pip freeze` and puts it into a file. Try running `pip freeze` without the `> requirements.txt` to see what happens!
+You can download Git from [git-scm.com](http://git-scm.com/).  You can hit "next next next" on all steps except for one; in the 5th step entitled "Adjusting your PATH environment", choose "Run Git and associated Unix tools from the Windows command-line" (the bottom option).  Other than that, the defaults are fine.  Checkout Windows-style, commit Unix-style line endings is good.
 
-Open this file and add the following line at the bottom:
+### MacOS
 
-    psycopg2==2.5.4
+Download Git from [git-scm.com](http://git-scm.com/) and just follow the instructions and just follow the instructions.
 
-This line is needed for your application to work on Heroku.
 
+### Linux
 
-## Procfile
+Most distros will have this packaged, so try
 
-Another thing we need to create is a Procfile. This will let Heroku know which commands to run in order to start our website.
-Open up your code editor, create a file called `Procfile` in `djangogirls` directory and add this line:
+    sudo apt-get install git  # on debian, ubuntu
+    # or
+    sudo yum install git  # on fedora etc
 
-    web: gunicorn mysite.wsgi
 
-This line means that we're going to be deploying a `web` application, and we'll do that by running the command `gunicorn mysite.wsgi` (`gunicorn` is a program that's like a more powerful version of Django's `runserver` command).
+## Starting our Git repository
 
-Then save it. Done!
+Go to your console and run these commands, in the `djangogirls` directory:
 
-## The `runtime.txt` file
+    $ git init
+    Initialized empty Git repository in ~/djangogirls/.git/
+    $ git config user.name "Your Name"
+    $ git config user.email you@example.com
 
-We need to tell Heroku which Python version we want to use. This is done by creating a `runtime.txt` in the `djangogirls` directory using your editor's "new file" command, and putting the following text (and nothing else!) inside:
+> __Note__: Check your current working directory with a `pwd` command before initializing the repository. You should be in the `djangogirls` folder.
```

Comment:
Actually, because we've chosen "run git and installed tools from the windows command-line" just earlier, `pwd` *will* work.  But, since you guys might choose to change that option in future, I'll add a windows-specific instruction.